financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundViewModel.kt

```diff
@@ -315,10 +318,18 @@ sealed class FinancialConnectionsPlaygroundViewEffect {
     ) : FinancialConnectionsPlaygroundViewEffect()
 }
 
-data class FinancialConnectionsPlaygroundState(
+internal data class FinancialConnectionsPlaygroundState(
     val backendUrl: String = "",
+    val settings: PlaygroundSettings,
```

Comment:
I still think keeping a mutable thing in something called state is weird. And I like the concept of the snapshot, but not blocking.

Reply:
`PlaygroundSettings` is slightly different in connections (it's a data class that wraps an inmutable map)